boards/xtensa/up_squared_adsp/tools/utilities/etrace.py

```diff
@@ -35,7 +39,7 @@ def __init__(self, dev, win_id=3, size=0x2000):
     def get_sram_win_offset(self, win_id):
         return (plat_def.FW_SRAM + (win_id * 0x20000))
 
-    def print(self):
+    def hexdump(self):
```

Comment:
This prevents the hexdump from the command. Either add new arg option for print hexdump (like if no output file is specified, print hexdump) or no need to process with Loglist here, since "logtool" can parse it.

Reply:
Yes, I have renamed print to hexdump, so that it is not get called since we do not need usually need it. So by default we call logtool and parse logs. Do you want to remove hexdump completely?

Reply:
I would like to keep it and will add arg options in dump_trace.py for hexdump once your PR is merged.